packages/mobile/src/account/FiatExchange.tsx

```diff
@@ -80,16 +77,26 @@ class FiatExchange extends React.Component<Props, State> {
   }
 
   render() {
-    return this.state.signedUrl === '' ? (
-      <ActivityIndicator size="large" color={colors.celoGreen} />
-    ) : (
-      <WebView style={styles.exchangeWebView} source={{ uri: this.state.signedUrl }} />
+    return (
+      <SafeAreaView style={styles.background}>
+        <DrawerTopBar />
+        {this.state.signedUrl === '' ? (
+          <ActivityIndicator size="large" color={colors.celoGreen} />
+        ) : (
+          <WebView style={styles.exchangeWebView} source={{ uri: this.state.signedUrl }} />
+        )}
+      </SafeAreaView>
     )
   }
 }
 
 const styles = StyleSheet.create({
   exchangeWebView: {},
+  background: {
+    backgroundColor: colorsV2.background,
```

Comment:
@aaronmgdr correct me if I am wrong, but should we be using `light` instead of `background`? Does it matter?

Reply:
technically background is an alias for `light` aka `#fff` now right now it logs that its deprecated but doesn't actually cause any harm. 

if we want to keep a `colors.background` we could remove the loging. I can see that being nice. 

thoughts ?

Reply:
I think `background` is more intuitive but have been using `light` everywhere because I saw the deprecation notice